src/Accounts/Accounts/Errors/ResolveError.cs

```diff
@@ -19,9 +19,11 @@
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
 using System.Collections;
 using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Profile.Errors
 {
+    [CmdletDeprecation(ChangeDescription = "Resolve-Error alias will be deprecated in the next breaking change release")]
```

Comment:
I am not sure what the output is when you run this cmdlet, can you verify and add to the comments?

Also, the message should contain a description of the change and the action the user should take:

```
The `Resolve-Error` alias will be removed in a future release.  Please change any scripts that use this alias to use `Resolve-AzError` instead.